<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CadastroUsuario</class>
 <widget class="QDialog" name="CadastroUsuario">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>668</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cadastrar Usuário</string>
  </property>
  <property name="locale">
   <locale language="Portuguese" country="Brazil"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1" rowspan="2">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="pushButtonCadastrar">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>Cadastrar</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonAtualizar">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>Atualizar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonBuscar">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../qrs/resources.qrc">
         <normaloff>:/search.png</normaloff>:/search.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonNovoCad">
       <property name="text">
        <string>Novo Cadastro</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonRemover">
       <property name="text">
        <string>Remover</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Cadastro Usuário</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="5" column="1">
        <widget class="QLineEdit" name="lineEditPasswd">
         <property name="accessibleName">
          <string>Senha</string>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 127);
color: rgb(0, 0, 0);</string>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="labelTipo">
         <property name="text">
          <string>Tipo:</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="TableView" name="table"/>
       </item>
       <item row="6" column="1">
        <widget class="QLineEdit" name="lineEditPasswd_2">
         <property name="accessibleName">
          <string>Conf. de senha</string>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 127);
color: rgb(0, 0, 0);</string>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="lineEditUser">
         <property name="accessibleName">
          <string>Nome de usuário</string>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 127);
color: rgb(0, 0, 0);</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="labelNome">
         <property name="text">
          <string>Nome completo:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="labelPasswd">
         <property name="text">
          <string>Senha:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="ComboBox" name="comboBoxLoja">
         <property name="accessibleName">
          <string>Loja</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="labelUser">
         <property name="text">
          <string>Usuário:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labelLoja">
         <property name="text">
          <string>Loja:</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="labelSigla">
         <property name="text">
          <string>E-mail:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEditNome">
         <property name="accessibleName">
          <string>Nome completo</string>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 127);
color: rgb(0, 0, 0);</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="comboBoxTipo">
         <property name="accessibleName">
          <string>Tipo</string>
         </property>
         <item>
          <property name="text">
           <string>VENDEDOR</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>ADMINISTRADOR</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>GERENTE FINANCEIRO</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>VENDEDOR ESPECIAL</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>GERENTE LOJA</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>GERENTE DEPARTAMENTO</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>DIRETOR</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>ADMINISTRATIVO</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="labelPasswd_2">
         <property name="text">
          <string>Conf. de Senha:</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QLineEdit" name="lineEditEmail">
         <property name="accessibleName">
          <string>Sigla</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="comboBoxEspecialidade">
         <item>
          <property name="text">
           <string>0 - Não definido</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1 - Revestimentos</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2 - Louças e Metais</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>3 - Acessórios</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>4 - Insumos</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>5 - Serviços</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="labelEspecialidade">
         <property name="text">
          <string>Especialidade:</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TableView</class>
   <extends>QTableView</extends>
   <header>src/tableview.h</header>
  </customwidget>
  <customwidget>
   <class>ComboBox</class>
   <extends>QComboBox</extends>
   <header>src/combobox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>lineEditNome</tabstop>
  <tabstop>comboBoxLoja</tabstop>
  <tabstop>comboBoxTipo</tabstop>
  <tabstop>comboBoxEspecialidade</tabstop>
  <tabstop>lineEditUser</tabstop>
  <tabstop>lineEditPasswd</tabstop>
  <tabstop>lineEditPasswd_2</tabstop>
  <tabstop>lineEditEmail</tabstop>
  <tabstop>table</tabstop>
  <tabstop>pushButtonCadastrar</tabstop>
  <tabstop>pushButtonAtualizar</tabstop>
  <tabstop>pushButtonBuscar</tabstop>
  <tabstop>pushButtonNovoCad</tabstop>
  <tabstop>pushButtonRemover</tabstop>
 </tabstops>
 <resources>
  <include location="../qrs/resources.qrc"/>
 </resources>
 <connections/>
</ui>
